gemfiles/rails5_1_4.gemfile

```diff
@@ -1,12 +1,14 @@
 source "https://rubygems.org"
 
+gem "rails", "5.1.4"
+gem "railties", "5.1.4"
 gem "activerecord", "5.1.4"
 gem "activesupport", "5.1.4"
 gem "actionview", "5.1.4"
-
 gem "mysql2", "~> 0.4.4"
 
-gem "minitest", "< 5.3.4"
-gem "mocha", "~> 0.14", require: false
+gem "nokogiri", "~> 1.10" if RUBY_VERSION.to_f < 2.5
```

Comment:
Why are these making changes?

Reply:
I asked to be listed in the DESCRIPTION.

Reply:
https://github.com/mixigroup/activerecord-turntable/pull/18#issuecomment-1609045294